src/capi.cpp

```diff
@@ -1098,6 +1098,74 @@ int rtcAddTrackEx(int pc, const rtcTrackInit *init) {
 	});
 }
 
+RTC_C_EXPORT int rtcAddAACTrack(int pc, const rtcTrackInit *init, uint8_t *latmMuxCfg, int size) {
```

Comment:
I would prefer that you add fields to `rtcTrackInit` rather than introducing a new function with duplicated code, just add a comment on them to specify that they are for AAC only.

Reply:
add a latmMuxCfg field for aac latm track.